src/models/Factor.js

```diff
@@ -185,13 +184,7 @@ function (Okta, Q, CookieUtil, factorUtil, BaseLoginModel) {
 
     save: function () {
       var rememberDevice = !!this.get('rememberDevice');
-      var username = this.appState.get('username');
       // Set/Remove the remember device cookie based on the remember device input.
-      if (rememberDevice) {
-        CookieUtil.setDeviceCookie(username);
-      } else {
-        CookieUtil.removeDeviceCookie();
-      }
```

Comment:
Since we don't use the device cookie anymore, we should also remove the device cookie methods from CookieUtil